src/uiConfigurations.js

```diff
@@ -266,7 +266,7 @@ function UiConfigs(props) {
                 {space !== "IPS"
                     ? <GoogleLogout
                         render={renderProps => (
-                            <Button variant="contained" startIcon={<ExitToAppIcon />} onClick={renderProps.onClick} disabled={renderProps.disabled}>Logout</Button>
+                            <Button variant="contained" startIcon={<ExitToAppIcon />} onClick={renderProps.onClick}>Logout</Button>
```

Comment:
Removing this prop is a workaround, but will remove certain features (disabling logout in case the user does not log in). This scenario will throw an error in case a user tries to log out.
Will you be able to explore this issue a little further, and find an alternate fix?